stdlib/public/runtime/SwiftRT-ELF.cpp

```diff
@@ -65,10 +67,22 @@ static void swift_image_constructor() {
       SWIFT_SECTION_RANGE(swift5_assocty),
       SWIFT_SECTION_RANGE(swift5_replace),
       SWIFT_SECTION_RANGE(swift5_replac2),
+      SWIFT_SECTION_RANGE(swift5_builtin),
+      SWIFT_SECTION_RANGE(swift5_capture),
   };
 
 #undef SWIFT_SECTION_RANGE
 
   swift_addNewDSOImage(&sections);
 }
 
+__asm__(".section \".note.swift_reflection_metadata\", \"aw\"");
+
+static __attribute__((__used__))
+__attribute__((__section__(".note.swift_reflection_metadata")))
+__attribute__((__aligned__(1)))
+struct {
+  const char MagicString[sizeof(SWIFT_REFLECTION_METADATA_ELF_NOTE_MAGIC_STRING)];
```

Comment:
I think that doing `sizeof(SWIFT_REFLECTION_METADATA_ELF_NOTE_MAGIC_STRING)/sizeof(*SWIFT_REFLECTION_METADATA_ELF_NOTE_MAGIC_STRING)` is better.